sdcflows/workflows/fit/syn.py

```diff
@@ -214,9 +215,14 @@ def init_syn_sdc_wf(
     find_zooms = pe.Node(niu.Function(function=_adjust_zooms), name="find_zooms")
     zooms_epi = pe.Node(RegridToZooms(), name="zooms_epi")
 
+    syn_config = data.load(f"sd_syn{'_sloppy' * sloppy}.json")
+
+    vox_params = pe.Node(niu.Function(function=_mm2vox), name="vox_params")
+    vox_params.inputs.registration_config = json.loads(syn_config.read_text())
```

Comment:
Since we need to read the config outside the Registration node, we might want to extract the number of levels from it (e.g., `len(config_dict['transforms'])` instead of hard-code the magic number (of 2). It will be a bit trickier for handling masks, but a bit more robust, IMHO.